miio/heater_miot.py

```diff
@@ -9,32 +9,66 @@
 from .miot_device import DeviceStatus, MiotDevice
 
 _LOGGER = logging.getLogger(__name__)
-_MAPPING = {
-    # Source https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:heater:0000A01A:zhimi-mc2:1
-    # Heater (siid=2)
-    "power": {"siid": 2, "piid": 1},
-    "target_temperature": {"siid": 2, "piid": 5},
-    # Countdown (siid=3)
-    "countdown_time": {"siid": 3, "piid": 1},
-    # Environment (siid=4)
-    "temperature": {"siid": 4, "piid": 7},
-    # Physical Control Locked (siid=6)
-    "child_lock": {"siid": 5, "piid": 1},
-    # Alarm (siid=6)
-    "buzzer": {"siid": 6, "piid": 1},
-    # Indicator light (siid=7)
-    "led_brightness": {"siid": 7, "piid": 3},
+_MAPPINGS = {
+    'zhimi.heater.mc2': {
+        # Source https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:heater:0000A01A:zhimi-mc2:1
+        # Heater (siid=2)
+        "power": {"siid": 2, "piid": 1},
+        "target_temperature": {"siid": 2, "piid": 5},
+        # Countdown (siid=3)
+        "countdown_time": {"siid": 3, "piid": 1},
+        # Environment (siid=4)
+        "temperature": {"siid": 4, "piid": 7},
+        # Physical Control Locked (siid=5)
+        "child_lock": {"siid": 5, "piid": 1},
+        # Alarm (siid=6)
+        "buzzer": {"siid": 6, "piid": 1},
+        # Indicator light (siid=7)
+        "led_brightness": {"siid": 7, "piid": 3},
+    },
+    'zhimi.heater.za2': {
+        # Source https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:heater:0000A01A:zhimi-za2:1
+        # Heater (siid=2)
+        "power": {"siid": 2, "piid": 2},
+        "target_temperature": {"siid": 2, "piid": 6},
+        # Countdown (siid=4)
+        "countdown_time": {"siid": 4, "piid": 1},
+        # Environment (siid=5)
+        "temperature": {"siid": 5, "piid": 8},
+        "relative_humidity": {"siid": 5, "piid": 7},
+        # Physical Control Locked (siid=7)
+        "child_lock": {"siid": 7, "piid": 1},
+        # Alarm (siid=3)
+        "buzzer": {"siid": 3, "piid": 1},
+        # Indicator light (siid=7)
+        "led_brightness": {"siid": 6, "piid": 1},
+    }
 }
 
 HEATER_PROPERTIES = {
-    "temperature_range": (18, 28),
-    "delay_off_range": (0, 12 * 3600),
+    'zhimi.heater.mc2': {
+        "temperature_range": (18, 28),
+        "delay_off_range": (0, 12 * 3600),
+    },
+    'zhimi.heater.za2': {
+        "temperature_range": (16, 28),
+        "delay_off_range": (0, 8 * 3600),
+    }
 }
 
 
 class LedBrightness(enum.Enum):
-    On = 0
-    Off = 1
+    """Xiaomi Smart Space Heater S (zhimi.heater.mc2): 
+    0 = On, 1 = Off
+
+    Xiaomi Smart Space Heater 1S (zhimi.heater.za2): 
+    0 = On (Bright), 1 = On (Dim), 2 = Off
```

Comment:
This is confusing API-wise.. How about just adding a `Dim = 2` to the original enum, and performing the conversion per-model basis (and warn about unsupported `Dim` for mc2?